src/xtopcom/xvm/xsystem_contracts/src/xtable_cross_chain_txs_collection_contract.cpp

```diff
@@ -28,10 +28,26 @@ void xtable_cross_chain_txs_collection_contract::setup() {
     // initialize map key
     STRING_CREATE(XPORPERTY_CONTRACT_PROCESSED_TABLE_BLOCK_HEIGHT);
     STRING_SET(XPORPERTY_CONTRACT_PROCESSED_TABLE_BLOCK_HEIGHT, std::to_string(0));
-    STRING_CREATE(XPORPERTY_CONTRACT_PROCESSED_LOGIC_TIME);
-    STRING_SET(XPORPERTY_CONTRACT_PROCESSED_LOGIC_TIME, std::to_string(0));
+  /*   STRING_CREATE(XPORPERTY_CONTRACT_PROCESSED_LOGIC_TIME);
+    STRING_SET(XPORPERTY_CONTRACT_PROCESSED_LOGIC_TIME, std::to_string(0));*/
 }
 
+void xtable_cross_chain_txs_collection_contract::updateCrossTx(std::string const& cross_txs_data) {
+    //  XCONTRACT_ENSURE(SOURCE_ADDRESS() == SELF_ADDRESS().to_string(), "xtable_cross_chain_txs_collection_contract instance is triggled by " + SOURCE_ADDRESS());
+    XCONTRACT_ENSURE(SELF_ADDRESS().to_string() == sys_contract_eth_table_cross_chain_txs_collection_addr,
+        "xtable_cross_chain_txs_collection_contract instance is not triggled by sys_contract_eth_table_cross_chain_txs_collection_addr");
+
+    if (!cross_txs_data.empty()) {
+        xstream_t stream(xcontext_t::instance());
+        stream << cross_txs_data;
+        CALL(common::xaccount_address_t { sys_contract_relay_make_block_addr }, "on_receive_cross_txs", std::string((char*)stream.data(), stream.size()));
+        xdbg("xtable_cross_chain_txs_collection_contract::on_timer,stream_size: %zu", stream.size());
+    }
+    auto const last_height = static_cast<std::uint64_t>(std::stoull(STRING_GET(XPORPERTY_CONTRACT_PROCESSED_TABLE_BLOCK_HEIGHT)));
+    STRING_SET(XPORPERTY_CONTRACT_PROCESSED_TABLE_BLOCK_HEIGHT, std::to_string(last_height + 1));
```

Comment:
change property name , XPORPERTY_CONTRACT_PROCESSED_TABLE_BLOCK_HEIGHT is not correct

Reply:
chang to XPORPERTY_CONTRACT_PROCESSED_TABLE_TX_COUNT